AESLib.c

```diff
@@ -64,6 +64,18 @@ void aes128_enc_single(const uint8_t* key, void* data){
 	aes128_enc(data, &ctx);
 }
 
+// encrypt multiple blocks of 128bit data, data_len but be mod 16
+// key is assumed to be 256bit thus 32 uint8_t's
```

Comment:
this seems to be a copy-and-paste victim? (the 256bit remark)

Reply:
yes, sorry about that